### PR TITLE
Rotate logs in production

### PR DESCRIPTION
**Pull request by afred**
Keeps 10 x 1 MB log files.

### CODE REVIEW

**Comment by afred** [comment]
@mccalluc?

File: config/environments/production.rb [hunk 1]
```diff
@@ -47,8 +47,8 @@
   # Prepend all log lines with the following tags.
   # config.log_tags = [ :subdomain, :uuid ]
 
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+  # Keep up to 10 x 1 MB log files for production
+  config.logger = config.logger = ActiveSupport::Logger.new(config.paths['log'].first, 10, 1_048_576)
```
Comment (mccalluc) [inline config/environments/production.rb]: `config.logger = config.logger =` ?

Comment (afred) [inline config/environments/production.rb]: geez. stand by..

Comment (afred) [inline config/environments/production.rb]: where were you on that one rubocop?

**Pull request merged**